src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractPackage.cs

```diff
@@ -2,39 +2,34 @@
 
 using System;
 using System.Threading;
-using System.Windows.Threading;
 using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
-using Microsoft.CodeAnalysis.Utilities;
+using Microsoft.VisualStudio.ComponentModelHost;
 using Microsoft.VisualStudio.Shell;
 using Task = System.Threading.Tasks.Task;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageService
 {
     internal abstract class AbstractPackage : AsyncPackage
     {
-        protected ForegroundThreadAffinitizedObject ForegroundObject;
+        protected ForegroundThreadAffinitizedObject ForegroundObject
+        {
+            get;
+            private set;
+        }
```

Comment:
nit: keep on single line.

Reply:
➡️ Leaving this for now to reduce code churn in light of https://github.com/dotnet/roslyn/pull/28781#discussion_r208562525 (where this property will be removed). Also related to https://github.com/dotnet/roslyn/pull/28781#discussion_r208566355.